### PR TITLE
Styling/movie summary

### PR DESCRIPTION
**Pull request by michaelputnam67**
## What does this PR do?
Add final styling for movie summary page and conditional rendering for tagline statistics and date. 

## Where should the reviewer start?
movie-summary.js

## How should this be manually tested?
Run npm test

### CODE REVIEW

**Review by sted1994 — APPROVED** [review]
Styling looks really good! Great job

File: src/Movie-Summary.js [hunk 1]
```diff
@@ -44,54 +45,59 @@ class MovieSummary extends Component {
...
+						Release Date
+					</span> {`${daysjs(this.state.movieData.release_date).format('MM-DD-YYYY')}` } </p> : 'N/A'}   
 					<p className='overview'>{this.state.movieData.overview}</p>
 				</div>
 				<ul className='statistics'>
 					<li> Average Rating {'  '}
+					{(this.state.movieData.average_rating) ? 
 						<NumberFormat
-							value={this.state.movieData.average_rating}
+							value={this.state.movieData.average_rating.toFixed(1)}
```
Comment (emmacbd) [inline src/Movie-Summary.js:62]: Thanks for fixing the nums.

File: src/Trailer.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import './css/trailer.css';
-import "react-responsive-carousel/lib/styles/carousel.min.css"
+import "react-responsive-carousel/lib/styles/carousel.min.css";
 import { Carousel } from 'react-responsive-carousel';
 
 const Trailer = ({trailers}) => {
 	return (
-		<Carousel ariaLabel='hello' infiniteLoop autoFocus={true} autoPlay={true} className='trailers-carousel' showThumbs={false}>
+		<Carousel showIndicators={(trailers.length <= 1) ? false : true} showStatus={false} infiniteLoop autoFocus={true} autoPlay={true} className='trailers-carousel' showThumbs={false}>
```
Comment (emmacbd) [inline src/Trailer.js:8]: Thoughtful UI touch, nice.

File: src/css/movie-summary.css [hunk 3]
```diff
@@ -6,8 +6,8 @@
 }
 
 .movie-summary {
-	background: black;
-	opacity: 80%;
+	background: radial-gradient(circle, #272525 1%, #0a0505a6 50%);
```
Comment (emmacbd) [inline src/css/movie-summary.css:9]: Looking radiant.

**Review by emmacbd — COMMENTED** [review]
Thanks for all the polishing!

**Pull request merged**